### PR TITLE
Players Tests

### PR DESCRIPTION
**Pull request by lucasgautheron**
Showing off my newbie skillzz : D (mostly copy paste from other tests here) Don't hesitate to tell me if it's useless.

BTW should tests rely on local data only ?

### CODE REVIEW

**Comment by ScalaWilliam** [comment]
Yup local only. Take a look here though

https://github.com/ScalaWilliam/ActionFPS/tree/master/accumulation/src/test/resources/af/samples

Plenty of samples.

File: players/src/main/scala/players/Players.scala [hunk 1]
```diff
@@ -162,9 +162,12 @@ case class PlayersStats(players: Map[String, PlayerStat]) {
     } yield user -> eloAddition
   }.toMap
 
+  def countElo(game: JsonGame): Boolean = {
+    game.teams.forall(_.players.forall(_.score.isDefined)) && game.teams.head.players.size == game.teams.last.players.size
```
Comment (ScalaWilliam) [inline players/src/main/scala/players/Players.scala]: Actually game.teams.map(_.size).toSet.size == 1 would be far better for the second predicate.

**Comment by ScalaWilliam** [comment]
There is an exception if you want to test against data that's a little bigger.

https://github.com/ScalaWilliam/ActionFPS/blob/master/accumulation/build.sbt#L5

But you don't need that method in this case.

File: players/src/test/scala/players/PlayersTest.scala [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import play.api.libs.json.Json
+
+/**
+  * Created by Lucas on 04/01/2016.
+  */
+class PlayersTest
+  extends FunSuite
+  with Matchers {
+  test("It should work") {
+    val jsn = Json.parse(new URL("http://api.actionfps.com/game/?id=2016-01-04T19:22:10Z").openConnection().getInputStream)
```
Comment (ScalaWilliam) [inline players/src/test/scala/players/PlayersTest.scala]: that needs to be a local file ;D

Comment (lucasgautheron) [inline players/src/test/scala/players/PlayersTest.scala]: I know don't worry. tests are just not my priority!

**Pull request merged**